pandas/_libs/groupby_helper.pxi.in

```diff
@@ -954,3 +954,59 @@ def group_shift_indexer(int64_t[:] out, int64_t[:] labels,
                     out[ii] = -1
 
                 label_indexer[lab, idxer_slot] = ii
+
```

Comment:
just realized this is in ``groupby_helper.pxi.in``, since this doesn't use templating (nor does group_shift_indexer). I think ok to omve to ``groupby.pxi``. can you move?